### PR TITLE
Add style

### PR DESCRIPTION
**Pull request by yosefHesham**
**Brief of what changed:**

Added styles to the bookstore to make it fancier and eye-catching.

---

**Changes Made:**

- Created progress component.
- Created chapter component.
- Created bookInfo component.

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,3 +1,197 @@
...
+}
+
+.author {
+  color: #4386bf;
+  font-weight: 300;
+  font-size: 0.875rem;
+}
+
+.actions {
+  display: flex;
```
Comment (ahmetbozaci) [inline src/App.css:95]: - [x] For DRY CSS you should combine duplicate properties. 
Example, you can create class `.d-flex { display: flex}` and use this class in HTML file.
### _This is just one example. Please check your CSS file and if some CSS rules are the same please combine them also._
- [More information about DRY principles](https://www.freecodecamp.org/news/7-important-tips-for-writing-better-css/) 

- [Html and CSS best practices in curriculum](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md#:~:text=If%20you%20want%20to%20create%20a%20CSS%20rule%20for%20multiple%20elements%2C%20use%20a%20class%20attribute%20instead.%20For%20example%2C%20instead%20of%20using%20color%3A%20%233498db%3B%203%20times)

File: src/components/navbar.js [hunk 2]
```diff
@@ -8,25 +8,31 @@ const Navbar = () => {
...
   ];
   return (
     <nav className="navBar" style={navStyle}>
-      <h1> Bookstore CMS </h1>
-      <ul style={{ ...navStyle, listStyle: 'none', columnGap: '2px' }}>
+      <ul
+        className="navList"
+        style={{ ...navStyle, listStyle: 'none', columnGap: '2px' }}
+      >
+        <h1> Bookstore CMS </h1>
```
Comment (ahmetbozaci) [inline src/components/navbar.js]: - [ ] The navigation bar is different than the original. To stick to the design kindly fix it.

image

___
ORIGINAL
image

File: src/components/add_book.js [hunk 3]
```diff
@@ -20,36 +27,50 @@ const AddBook = () => {
...
+          name="author"
+          placeholder="Book author"
+          value={author}
+          onChange={handleChange}
+        />
+        <button type="submit" className="input-submit">
+          Submit
+        </button>
+      </form>
+    </>
```
Comment (ahmetbozaci) [inline src/components/add_book.js]: - [ ] The form is different from the original. It is ok if you add author input but please add category input also.
- [ ] Please align the form to the book list.
- [ ] Please change the update progress button style like in the original.

image
__
Original
image

**Review by ahmetbozaci — CHANGES_REQUESTED** [review]
# STATUS: Changes Required  ♻️


 @yosefHesham  ✋🏼 

#### Your project is great. You are almost done. However, there are some issues you should fix. Please fix them then your project will be approved.

___ 

### 💯  Nice to have 💯
- Your git-flow is correct. ✔️
- You set up linter files and no errors. ✔️
- Your pull request is very professional. It has a meaningful title and summary. Keep it up.✔️
___


### ♻️ Request changes ♻️

- [ ] _Check the comments under the review._
- [ ] Kindly fix the broken link in the license section.
 image


___

### Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
___

### Cheers and Happy coding! 💻 👏
almost


___

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification._

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._


___

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)_

[GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)

_Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

**Review by zieeco — APPROVED** [review]
# status: APPROVED :green_circle: :100: 

Hi, @yosefHesham , 


68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f78543078657a514755357843444a754350652f67697068792e676966


Good job so far 👍 Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights ⚡
 
✅ Descriptive PR 👍 
✅ Nice design 🔥 
✅ Good commit messages 👍 
✅ Linters are passing 👏 


## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**